ui/ui.js

```diff
@@ -247,8 +247,10 @@ shaka.ui.Overlay = class {
       fullScreenElement: this.videoContainer_,
     };
 
-    // Check AirPlay support
-    if (window.WebKitPlaybackTargetAvailabilityEvent) {
+    // eslint-disable-next-line no-restricted-syntax
+    if ('remote' in HTMLMediaElement.prototype) {
```

Comment:
Is this check correct? It looks like the `remote` property is present on video elements in desktop chrome, but based on our earlier discussion the RemotePlayback API doesn't work properly there, right?

Reply:
I check if the API is available or not, and then I use a logic to detect if it works or not based on the availability.